scripts/typescript_checks.py

```diff
@@ -61,38 +61,30 @@ def compile_and_check_typescript(config_path):
         config_path: str. The config that should be used to run the typescript
             checks.
     """
-    node_path = common.NODE_PATH
-    os.environ['PATH'] = '%s/bin:' % node_path + os.environ['PATH']
-
+    os.environ['PATH'] = '%s/bin:' % common.NODE_PATH + os.environ['PATH']
     validate_compiled_js_dir()
 
     if os.path.exists(COMPILED_JS_DIR):
         shutil.rmtree(COMPILED_JS_DIR)
 
     print('Compiling and testing typescript...')
-    cmd = [
-        './node_modules/typescript/bin/tsc', '--project',
-        config_path]
-    process = subprocess.Popen(
-        cmd, stdout=subprocess.PIPE, encoding='utf-8')
-    error_messages = []
-    for line in iter(process.stdout.readline, ''):
-        if not line.startswith('node_modules'):
```

Comment:
ANy specific reason for this change?

Reply:
It is not needed anymore, I changed the tsconfig.json in a way that errors for files in node_modules are no longer produced.